src/CredentialsLoader.php

```diff
@@ -247,4 +250,52 @@ public function updateMetadata(
 
         return $metadata_copy;
     }
+
+    public static function getDefaultClientCertSource()
+    {
+        if (!$clientCertSourceJson = self::loadDefaultClientCertSourceFile()) {
+            return null;
+        }
+        if (!isset($clientCertSourceJson['cert_provider_command'])) {
```

Comment:
WDYT about putting this validation in the body of `loadDefaultClientCertSourceFile`?

Reply:
done